.github/workflows/containers-ghcr.yml

```diff
@@ -0,0 +1,79 @@
+name: "Build and Publish Release Containers to GHCR"
+
+on:
+  release:
+    types: [published]
+
+env:
+  REGISTRY: ghcr.io
+
+permissions: read-all
+
+jobs:
+  build_and_publish:
+    name: "Build and Publish Release Containers"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [sgx, virtual, snp]
+        type: [dev, run]
+        run_js: [true, ""]
```

Comment:
Why is `run_js` a separate option here, rather than a value in `type`? eg - `type: [dev, run, "run-js"]`

Reply:
Because having it as a bbol allows inline conditions for different values, which we use for `${{ matrix.run_js && 'run_js=true' || '' }}`. We could also do that with an action level `if:`, but we would need to duplicate the whole action. I am open to better ideas though.